builds/make.type.disk

```diff
@@ -36,12 +36,11 @@ DFLAGS    += -DTEMPERATURE_FLOOR
 DFLAGS    += -DCOOLING_GPU
 #DFLAGS    += -DCLOUDY_COOL
 DFLAGS    += -DDE
-#DFLAGS    += -DCPU_TIME
+DFLAGS    += -DCPU_TIME
 DFLAGS    += -DAVERAGE_SLOW_CELLS
 DFLAGS    += -DHYDRO_GPU
 
-OUTPUT    ?=  -DOUTPUT -DHDF5
-# -DSLICES
+OUTPUT    ?=  -DOUTPUT -DHDF5 -DSLICES -DPROJECTION
```

Comment:
Should we be outputting slices and projections by default?

Reply:
My opinion is that slices and projections are cheap and more frequently needed than not for the disk problem, so defaulting to output is better. Especially to minimize the number of manual edits and changes needed to run production.

Reply:
Yeah, I think it's fine to have these on since everyone who runs disk problems uses them.